setup.cfg

```diff
@@ -34,6 +34,7 @@ zip_safe = False
 install_requires =
     six>=1.10
     isbnlib>=3.10.8
+    importlib-metadata>=6.11.0
```

Comment:
this should not be needed if we have Python >= 3.9:

```shell
~/invenio/modules/idutils > python --version
Python 3.9.16
~/invenio/modules/idutils > python
Python 3.9.16 (main, Oct 12 2023, 17:53:07)
[Clang 15.0.0 (clang-1500.0.40.1)] on darwin
Type "help", "copyright", "credits" or "license" for more information.
>>> from importlib.metadata import entry_points
>>>
```

Reply:
but it is needed for python >= 3.12

Reply:
you are right. also for python >= 3.12 your line works. BUT the idutils is using importlib_metadata: https://github.com/inveniosoftware/idutils/blob/812f640ee78c4f3f52f7dcef6c91ae747f100d50/idutils/ext.py#L53 and not importlib.metadata

```
Python 3.12.7 (main, Oct  2 2024, 16:29:41) [GCC 12.2.1 20230304] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> from importlib_metadata import entry_points
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
ModuleNotFoundError: No module named 'importlib_metadata'
>>> 
```

Reply:
using `importlib.metadata` is possible for python3.9 because it has been added with python3.8: https://docs.python.org/3/library/importlib.metadata.html

Reply:
I see that you have answered yourself :D

I asked @ptamarit if he could create a PR with:
- changing the imports to `importlib.metadata`
- changing the publish CI to use Python 3.9
- remove any install dependency of `importlib`
and notify everyone, but @tmorrell was too fast :)

I believe the above should work and can be applied to other modules when touched/needed.

Reply:
> I tried importlib.metadata and it didn’t work exactly the same. That’s not to say it couldn’t be used…but it’s not a drop in replacement

Yes you might be right. The same change was done [here](https://github.com/inveniosoftware/pywebpack/pull/44/files#diff-0fa85cbd790a9c260987adf0bc1d57dc10c426e0479421a95306508636db1057R34), and tested it.
Is it probably this one to apply?